angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -162,14 +173,21 @@ import { ISprkDropdownChoice } from './sprk-dropdown.interfaces';
 })
 export class SprkDropdownComponent implements OnChanges {
   /**
-   * The variant of the Dropdown to render.
+   * Determines the type of Dropdown to render.
    */
   @Input()
-  dropdownType = 'base';
+  variant: string;
+  // TODO: #3800 Remove `dropdownType` input, now replaced with `variant`
+  /**
+   * Deprecated: Use the `variant` input instead.
```

Comment:
Let's update this to 'Deprecated. Use the `variant` input instead.' For consistency.